cmd/terramate/cli/cloud.go

```diff
@@ -78,12 +78,41 @@ func (c *cli) checkSyncDeployment() {
 		fatal(err)
 	}
 
-	if orgs := c.cred().organizations(); len(orgs) != 1 {
+	// at this point we know user is onboarded, ie has at least 1 organization.
+	orgs := c.cred().organizations()
+
+	useOrgName := os.Getenv("TM_CLOUD_ORGANIZATION")
+	if useOrgName != "" {
+		var useOrgUUID string
+		for _, org := range orgs {
+			if org.Name == useOrgName {
+				if org.Status != "active" {
+					fatal(errors.E("You are not yet an active member of organization %s. Please accept the invitation first.", useOrgName))
+				}
+
+				useOrgUUID = org.UUID
+				break
+			}
+		}
+
+		if useOrgUUID == "" {
+			fatal(errors.E("You are not a member of organization %q or the organization does not exist. Available organizations: %s",
+				useOrgName,
+				orgs,
+			))
+		}
+
+		c.cloud.run.orgUUID = useOrgUUID
+	} else if len(orgs) != 1 {
 		fatal(
-			errors.E("requires 1 organization associated with the credential but %d found: %s",
-				len(orgs),
-				orgs),
+			errors.E("Please set TM_CLOUD_ORGANIZATION environment variable to a specific available organization %s", orgs),
```

Comment:
Suggestion: We should consider in the future to make this configurable via files, and not just environment variables. As it is exceedingly unlikely for anyone to run into this right now, this is not actually a problem.

Reply:
for sure it'd be great to maybe also support a command line flag.